src/component/PricingButton/PricingButton.jsx

```diff
@@ -24,10 +26,19 @@ async function getPrices({
   const response = await axios
     .get(pricesEndpoint, params)
     .then((res) => {
+      
       return res.data;
     })
-    .catch(() => {
-      return false;
+    .catch((error) => {
+      const error_response = {
+        error: true,
+        status: error.response.status,
+        message: {
+          lyft: undefined,
+          uber: undefined,
+        },
+      };
+      return error_response;
```

Comment:
This is return statement is redundant. You can simply return the object. 
```suggestion
    .catch((error) => {
      return {
        error: true,
        status: error.response.status,
        message: {
          lyft: undefined,
          uber: undefined,
        },
      };
```